src/router.ts

```diff
@@ -57,5 +57,13 @@ export default new Router({
       // which is lazy-loaded when the route is visited.
       component: () => import(/* webpackChunkName: "about" */ './views/MintRealEstate.vue'),
     },
+    {
+      path: '/mint/video-latino',
+      name: 'mint-video-latino',
+      // route level code-splitting
+      // this generates a separate chunk (about.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
+      component: () => import(/* webpackChunkName: "about" */ './views/MintVideoLatino.vue'),
```

Comment:
We should split the chunks up - `webpackChunkName: "about"`

Reply:
Not sure how this works - will need to read up about it